src/main/java/com/xiaomi/infra/pegasus/example/ExampleSample.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+package com.xiaomi.infra.pegasus.example;
+
+import com.xiaomi.infra.pegasus.client.*;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+
+/** Example sample */
+public class ExampleSample {
+  public static void main(String[] args) throws PException, UnsupportedEncodingException {
+    PegasusClientInterface client = PegasusClientFactory.createClient(ClientOptions.create());
+    PegasusTableInterface table = client.openTable("temp");
+
+    byte[] value = new byte[1000];
+
+    for (int i = 0; i < 1000; i++) {
+      value[i] = 'x';
+    }
+
```

Comment:
Long value can be generate by `RandomStringUtils.random`

Reply:
yeah! Got it